test/config/TestConfig.cpp

```diff
@@ -533,3 +557,47 @@ TEST(Config, PubSubSampleCli)
     ASSERT_STREQ(filePath.c_str(), config.pubSub.subscribeFile->c_str());
     remove(filePath.c_str());
 }
+
```

Comment:
Add unit tests for Serializing to JSON object.

Reply:
have added that in TestConfig